### PR TITLE
KFSPTS-26979 Fix tax PDP handling for chart-to-campus rename

### PR DESCRIPTION
**Pull request by cah292**
NOTE: I'm still wrapping up local validation of this. Please don't merge until I add a comment indicating it's safe to do so.

One of the recent KualiCo patches renamed the PDP Customer Profile's Chart field to Campus instead, but our custom Tax Processing Job did not get updated yet to compensate for that. This PR fixes the issue by making the following changes:

* In the PDP data query that still needs to reference the renamed field, that section and the related areas have been updated to correctly reference the new Campus field.
* In the section that validates the account and org data from the PDP transaction, that section has been updated to derive the Chart from the payment's accounting details instead of from the renamed field. (The functionals have requested this change; see the user story.) This also allowed for removing a hack that temporarily stored the old renamed field inside one of the other data fields.

### CODE REVIEW

File: src/main/java/edu/cornell/kfs/tax/dataaccess/impl/TransactionRowPdpBuilder.java
Comment (jhulslander) [inline src/main/java/edu/cornell/kfs/tax/dataaccess/impl/TransactionRowPdpBuilder.java:285]: Just to confirm, we want the document title to be null?

**Review by jhulslander — COMMENTED** [review]
Changes look good, I just had one question

File: src/main/java/edu/cornell/kfs/tax/dataaccess/impl/TransactionRowPdpBuilder.java
Comment (cah292) [reply src/main/java/edu/cornell/kfs/tax/dataaccess/impl/TransactionRowPdpBuilder.java:285]: If I remember correctly, the tax processing code does two passes for setting up the transaction detail rows. On the first pass, if the main data setup method doesn't explicitly set a value for a given field, then this `insertNullsForTransactionRow()` method needs to explicitly set those unset fields to null so that the SQL insert query will succeed. The code for the second pass should set these fields (including the doc title) to non-null values as needed.

Comment (jhulslander) [reply src/main/java/edu/cornell/kfs/tax/dataaccess/impl/TransactionRowPdpBuilder.java:285]: OK, thanks

**Comment by cah292** [comment]
I've added a change to fix another issue that I discovered in local testing:

* Fixed a section of the code that was hard-coded to reference the KRNS_DOC_HDR_T table, which was consolidated into FS_DOC_HEADER_T in a recent KFS upgrade. The section has been updated to point to FS_DOC_HEADER_T instead; we can refactor this further in our future revisions to this batch job.

I'm running another quick local test to verify that this fix works and that there are no further issues.

**Comment by cah292** [comment]
My local run of the job in 1099 mode succeeded with the changes above in place. This is ready to merge once approved.

File: src/main/java/edu/cornell/kfs/tax/dataaccess/impl/TaxSqlUtils.java
Comment (ajd299) [inline src/main/java/edu/cornell/kfs/tax/dataaccess/impl/TaxSqlUtils.java:519]: Does the SQL need a schema reference? (kfs.)

Comment (cah292) [reply src/main/java/edu/cornell/kfs/tax/dataaccess/impl/TaxSqlUtils.java:519]: It ran fine without it, but I can add it and then we can validate further in DEV.

Comment (ajd299) [reply src/main/java/edu/cornell/kfs/tax/dataaccess/impl/TaxSqlUtils.java:519]: okay given that the schema reference wasn't there before I will merge.

**Review by ajd299 — APPROVED** [review]
Code looks good, merging

File: src/main/java/edu/cornell/kfs/tax/dataaccess/impl/TaxSqlUtils.java
Comment (cah292) [reply src/main/java/edu/cornell/kfs/tax/dataaccess/impl/TaxSqlUtils.java]: Schema reference has been added. Further cleanup of schema references in this process will be handled in separate tickets as appropriate.

**Pull request merged**
